.github/workflows/build.yml

```diff
@@ -40,4 +46,4 @@ jobs:
         if: ${{ matrix.java_version != '11' }}
 
       - name: Build with Gradle
-        run: ./gradlew build --stacktrace -DtestConfigMethod=${{ matrix.test_config_method }}
+        run: ./gradlew build --stacktrace -DtestJavaVersion=${{ matrix.java_version }} -DtestConfigMethod=${{ matrix.test_config_method }}
```

Comment:
I believe we have no need to add this, dropping 11 from the test matrix is enough. https://github.com/vanniktech/gradle-maven-publish-plugin/blob/4e2970e02f065e6545ec3df7eb79cf5d481a8036/.github/workflows/build.yml#L20

Java 11 level has already been pinned by #1145.

Reply:
I want to keep running the tests on 11 as long as we support Gradle 8.x.